prov/efa/src/efa_strerror.c

```diff
@@ -101,18 +125,100 @@ static const char *efa_errno_str(enum efa_errno err)
 		: errno_str[err - FI_EFA_ERRNO_OFFSET];
 }
 
+static const char *efa_debug_str(enum efa_errno err)
+{
+	switch (err) {
+	case FI_EFA_REMOTE_ERROR_BAD_DEST_QPN:
+		return "This error can be caused by a crashed peer. "
+			   "Please verify the receiver status";
+	case FI_EFA_LOCAL_ERROR_UNRESP_REMOTE:
+		return "This error can be caused by a crashed peer or "
+			   "incrorrect inbound/outbound rules in the security group - "
+			   "EFA requires \"All traffic\" type allowlisting. "
+			   "Please also verify the receiver status.";
```

Comment:
maybe a little more explicit than "peer status":
"Please also verify the peer is still healthy and has not terminated unexpectedly"